tests/Rector/MethodCall/SwiftCreateMessageToNewEmailRector/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Rector\Symfony\Tests\Rector\MethodCall\SwiftCreateMessageToNewEmailRector\Fixture;
+
+class Fixture
+{
+    public function __construct(\Swift_Mailer $swift)
```

Comment:
I was hoping that rector would be able to detect that `$this->swift` will be of type `\Swift_Mailer` in order to be a little safer when refactoring but this doesn't seem to be the case, did I miss something for that?

Reply:
In some cases PHPStan needs to have stubs in `/stubs` directory.